### PR TITLE
update go version

### PR DESCRIPTION
**Pull request by shibuiwilliam**
# What
- update go version to 1.18

### CODE REVIEW

**Comment by Konboi** [comment]
We need to update from `go get` to `go install` to install commands 
https://github.com/launchableinc/examples/blob/master/.github/workflows/go.yml#L40 

ref: https://go.dev/doc/go-get-install-deprecation#:~:text=Starting%20in%20Go%201.17%2C%20installing,the%20%2Dd%20flag%20were%20enabled.

**Comment by shibuiwilliam** [comment]
> We need to update from `go get` to `go install` to install commands https://github.com/launchableinc/examples/blob/master/.github/workflows/go.yml#L40
> 
> ref: https://go.dev/doc/go-get-install-deprecation#:~:text=Starting%20in%20Go%201.17%2C%20installing,the%20%2Dd%20flag%20were%20enabled.

@Konboi 
Thank you. Fixed

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -35,9 +35,11 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v2
         with:
-          go-version: 1.17
+          go-version: 1.18
       - name: Install dependencies
-        run: go get -u github.com/jstemmer/go-junit-report
+        run: |
+          go get github.com/jstemmer/go-junit-report
+          go install github.com/jstemmer/go-junit-report
```
Comment (Konboi) [inline .github/workflows/go.yml:42]: How about this?

```suggestion
          go install github.com/jstemmer/go-junit-report@latest
```

Comment (shibuiwilliam) [reply .github/workflows/go.yml]: fixed
https://github.com/launchableinc/examples/pull/45/commits/482cc044c6532b08c322166269d71e91c98049c2

**Review by Konboi — APPROVED** [review]
LGTM

**Pull request merged**